pkg/controller/networkpolicy/antreanetworkpolicy.go

```diff
@@ -24,19 +24,22 @@ import (
 	antreatypes "antrea.io/antrea/pkg/controller/types"
 )
 
+func getAntreaNetworkPolicyReference(anp *crdv1alpha1.NetworkPolicy) controlplane.NetworkPolicyReference {
```

Comment:
`internalNetworkPolicyQueue` must use value instead of pointer as the key, otherwise the same NetworkPolicies will not be treated as same item because the pointers may be different. But other places could use pointers to save some copy overhead. I have updated them.